docs/roadmap.rst

```diff
@@ -51,11 +51,13 @@ One metric of success, then, is to develop documentation that includes:
 
 .. _a ReadTheDocs site: https://tedana.readthedocs.io
 
-**Associated Milestone**:
+**Associated Milestone**: `#6`_
```

Comment:
Great call ! But the links aren't going to play nicely with the pound signs. Could we actually just wrap it s.t. the links each look like 
```rst
**`Associated Milestone`_**

.. _Associated Milestone: https://github.com/ME-ICA/tedana/milestone/6
```
?

Reply:
Yep - see - I knew I didn't know the `rst` format well enough! I'll make these changes now 😸